<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Կոնդենսացիայի գրաֆի և ՈՒԿԿ վիզուալիզատոր</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-size: 15px;
	font-weight: 600;
}

QMainWindow {
    background-color:  #f9f9f9;
}

QToolBar, QMenuBar {
    background-color: #f9f9f9;
    border: 1px solid #e1e1e1;
}

QMenuBar::item:selected, QToolBar::item:hover {
    background-color: #f0f0f0;
}

QPushButton {
    background-color: #ffffff;
    color: #333333;
    border: 2px solid #cccccc;
    border-radius: 4px;
    padding: 5px;
}

QPushButton:hover {
    border-color: #0078d7;
}

QPushButton:pressed {
    background-color: #cccccc;
}

QTextEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 4px;
}

QStatusBar {
    background-color: #ffffff;
    color: #333333;
}

QScrollArea {
        border: none;
    }

    QScrollBar {
        background: white;
        border-radius: 5px;
    }

    QScrollBar:horizontal {
        height: 13px;
    }

    QScrollBar:vertical {
        width: 13px;
    }

    QScrollBar::handle {
        background: #3d3d3d;
        border-radius: 5px;
    }

    QScrollBar::handle:horizontal {
        height: 25px;
        min-width: 10px;
    }

    QScrollBar::handle:vertical {
        width: 25px;
        min-height: 10px;
    }

    QScrollBar::add-line {
        border: none;
        background: none;
    }

    QScrollBar::sub-line {
        border: none;
        background: none;
    }
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="graphArea" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="85,15">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="GraphWidget" name="graphicsView">
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="text2Graph" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>24</number>
          </property>
          <property name="bottomMargin">
           <number>24</number>
          </property>
          <item>
           <widget class="QLabel" name="nodeCountLabel">
            <property name="text">
             <string>Գագաթների քանակ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nodeCountText">
            <property name="styleSheet">
             <string notr="true">border: 1px solid black;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="edgesLabel">
            <property name="text">
             <string>Կողեր</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="edgesText">
            <property name="styleSheet">
             <string notr="true">border: 1px solid black;
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="graphGenerationButton">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: black;
    color: white;
    border: 2px solid white;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
    text-align: center;
    outline: none;
}

QPushButton:hover {
    background-color: #333333; 
    border-color: #f2f2f2;
    text-decoration: none;
}

QPushButton:pressed {
    background-color: #1a1a1a; 
    border-color: #e6e6e6; 
}
</string>
            </property>
            <property name="text">
             <string>Գեներացնել</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Ֆայլ</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Փոփոխել</string>
    </property>
    <addaction name="actionForce"/>
    <addaction name="actionDraw"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionTranspose"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Ալգորիթմներ</string>
    </property>
    <addaction name="actionSCC"/>
    <addaction name="actionCondensationGraph"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar {
    spacing: 20px; 
}

QToolBar &gt; QToolButton {
    margin-left: 5px;
    margin-right: 5px;
	margin-top: 1px;
}

QToolBar &gt; QToolButton:hover {
    background-color: #f0f0f0;	
	border: 0px;
}
</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionForce"/>
   <addaction name="actionDraw"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionTranspose"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionSCC"/>
   <addaction name="actionCondensationGraph"/>
   <addaction name="actionVisualize"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Բացել</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Պահպանել</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/clear.png</normaloff>:/images/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Մաքրել</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/delete.png</normaloff>:/images/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Ջնջել</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/draw.png</normaloff>:/images/images/draw.png</iconset>
   </property>
   <property name="text">
    <string>Նկարել</string>
   </property>
  </action>
  <action name="actionForce">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/move.png</normaloff>:/images/images/move.png</iconset>
   </property>
   <property name="text">
    <string>Շարժել</string>
   </property>
  </action>
  <action name="actionTranspose">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/transpose.png</normaloff>:/images/images/transpose.png</iconset>
   </property>
   <property name="text">
    <string>Շրջել</string>
   </property>
  </action>
  <action name="actionCondensationGraph">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/convert.png</normaloff>:/images/images/convert.png</iconset>
   </property>
   <property name="text">
    <string>Խտացնել</string>
   </property>
  </action>
  <action name="actionSCC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/scc.png</normaloff>:/images/images/scc.png</iconset>
   </property>
   <property name="text">
    <string>ՈՒԿԿ</string>
   </property>
  </action>
  <action name="actionVisualize">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/visualize.png</normaloff>:/images/images/visualize.png</iconset>
   </property>
   <property name="text">
    <string>Ալգորիթմ</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QGraphicsView</extends>
   <header>graphwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
